en/developer_guide/custom_content_attribute_display.rst

```diff
@@ -0,0 +1,54 @@
+Customize Content Attribute Display
+===================================
+
+Context
+-------
+
+In order to have minimum constraints on contents, Open Orchestra allows you to store any type
+of ``value`` in the ``ContentAttribute``.
+Sometimes, for instance in data tables, there is no easy way to render the value to the backoffice user so we have added
+a ``stringValue`` to ``ContentAttribute``.
+The ``stringValue`` is a projection of the ``value`` in a string.
+
+Transformers
+------------
+
+Adding a transformer
+~~~~~~~~~~~~~~~~~~~~
+
+Open Orchestra transforms the value into a string thanks to strategy implementing the interface
```

Comment:
I don't like this sentence. I would rather turn it another way by saying : Implementing the ... interface and adding the ... tag will allow open-orchestra to transform the value .